doc/Developing.md

```diff
@@ -44,7 +44,7 @@ For Windows instructions [click here](#building-on-windows).
      - [Root privileges errors](#root-privileges-errors)
 
 ## Prerequisites
- - Node.js v8.0 or higher
+ - Node.js v8.0+ or higher. Node.js 10 is **not** supported yet due to an issue in [`nsfw`](https://github.com/theia-ide/theia/issues/2009). Preferably, use version `8.11.4`, that is the currently [active LTS](https://github.com/nodejs/Release).
```

Comment:
Only at runtime, at development time in the case of Electron it cannot be higher than Node.js compatible with Electron 1.8.2. I don't really know which version it is, some 8.x.x.

Reply:
It works for me with `8.11.4`, that's why the recommendation.

Reply:
ok, I just don't think it should be a recommendation, extension developers should not use anything higher than Node.js 8 till we migrate to Electron 3

Using higher versions only work for deployment of browser apps.

Reply:
> I just don't think it should be a recommendation

> extension developers should not use anything higher than Node.js 8

`8.11.4` is not higher than 8.

Or I do not understand what do you want. Just tell me what do you want to see in the docs and I do the update. Otherwise, please approve. Thanks!

Reply:
My main concern is `+ or higher`. I think, for development prerequisites should be something like `Node.js >= 8.x, < 9.0.0`. In reality even 8.11.4 is not good, because it has things like `util.promisify` which are not available in Electron 1.8.2. We need latest Electron 2.x to support latest Node.js 8.x.

Reply:
I see. OK, I thought you have problems with the version `8.11.4`, I did not get it. Thanks for clarifying. I will make the change.